blender/src/models/home.rs

```diff
@@ -0,0 +1,69 @@
+use super::category::BlenderCategory;
+use crate::page_cache::PageCache;
+use regex::Regex;
+use std::io::Result;
+use url::Url;
+
+#[derive(Debug)]
+pub struct BlenderHome {
+    // might use this as a ref?
+    // TODO: Find out how I can sort this via original list collection.
+    // sort by descending order
+    // list.sort_by(|a, b| b.cmp(a));
+    list: Vec<BlenderCategory>,
+    // I'd like to reuse this component throughout blender program. If I need to access a web page, this should be used.
+    cache: PageCache,
+}
+
+impl BlenderHome {
+    fn get_content(cache: &mut PageCache) -> Result<Vec<BlenderCategory>> {
+        let parent = Url::parse("https://download.blender.org/release/").unwrap();
+        let content = cache.fetch(&parent)?;
+
+        // Omit any blender version 2.8 and below
+        let pattern = r#"<a href=\"(?<url>.*)\">(?<name>Blender(?<major>[3-9]|\d{2,}).(?<minor>\d*).*)\/<\/a>"#;
+        let regex = Regex::new(pattern).unwrap();
```

Comment:
For this initial phase unwrapping is ok, but consider adding some error handling later.

Reply:
Hi Luis, I can't think of a scenario where this regex would fail parsing the pattern. This was used to obtain information from a webpage - but would like to know if there's a better strongly typed way to search through HTML code?

Reply:
Guess it's simpler to use unwrap, but if this operation is done only once you could make it infallible by using a lazy initialization.